### PR TITLE
=doc explain parallelism-max in more detail

### PR DESCRIPTION
**Pull request by ktoso**
Wording maybe needs to be improved, but found that parallelism is confusing people.

// cc @viktorklang (it's related to the thing we worked on yesterday)

### CODE REVIEW

File: akka-docs/rst/java/dispatchers.rst [hunk 1]
```diff
@@ -34,6 +34,12 @@ is to configure the dispatcher:
 
 .. includecode:: ../scala/code/docs/dispatcher/DispatcherDocSpec.scala#my-dispatcher-config
 
+.. warning::
```
Comment (viktorklang) [inline akka-docs/rst/java/dispatchers.rst]: IMO it's not a warning, it's an explanation. :)

**Comment by viktorklang** [comment]
:+1:

File: akka-docs/rst/java/dispatchers.rst (hunk 1 above)
Comment (ktoso) [inline akka-docs/rst/java/dispatchers.rst]: True, I'll make it a note :)

File: akka-docs/rst/scala/dispatchers.rst [hunk 2]
```diff
@@ -34,6 +34,12 @@ is to configure the dispatcher:
 
 .. includecode:: ../scala/code/docs/dispatcher/DispatcherDocSpec.scala#my-dispatcher-config
 
+.. warning::
+  Note that the ``parallelism-max`` does not set the upper bound on the total number of threads
```
Comment (analytically) [inline akka-docs/rst/scala/dispatchers.rst]: Mention the default value perhaps?

Comment (viktorklang) [inline akka-docs/rst/scala/dispatchers.rst]: The default value depends on the JDK version.

**Comment by johanandren** [comment]
LGTM with the exception of Viktors note about warning -> note

**Pull request merged**